public/js/serchDir/searchDir.js

```diff
@@ -0,0 +1,83 @@
+const affiliateId = 789876;
```

Comment:
I also renamed a few variables in there to make it easier for myself to understand what was happening.